### DISCUSSION TITLE
DeepMD in LAMMPS with atoms of different types.

### DISCUSSION THREAD

**Discussion by rohskopf** [deepmodeling/deepmd-kit#207] [Q&A]
I made a DeepMD potential for a system containing aluminum (Al) and oxygen (O). In the type.raw file, aluminum was type "0" and oxygen was "1". 

The potential works fine, but now I want to do a LAMMPS calculation where I have multiple Al atoms, but each have a different "type" in LAMMPS. For example, some Al atoms are type "1" while others are type "2".

How would this work in DeepMD. In other words, how does DeepMD realize that one atom is a particular species regardless of its type?

**Comment by th-nic** [comment]
Did you find any solution to this issue? Or is there any? I want to assign one of the H-atom with a mass 1 u while the other as 2 u.
